ElectronClient/gui/style/theme/nord.js

```diff
@@ -77,7 +77,7 @@ const nordStyle = {
 	htmlCodeBorderColor: nord[2],
 	htmlCodeColor: nord[13],
 
-	editorTheme: 'chaos',
+	editorTheme: 'terminal',
```

Comment:
In AceEditor.tsx, you've added the "tomorrow" theme, and "twilight" was already present, however you didn't add "terminal". Isn't it an issue?

Reply:
yeah thats my bad i apologize for that. ive added it in the new commit.